docs/conceptual/option.filter['t]-function-[fsharp].md

```diff
@@ -0,0 +1,63 @@
+---
+title: Option.filter<'T> Function (F#)
+description: Option.filter<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: dend
```

Comment:
Because this is an entirely new article, please make sure that the `author:` field represents you 😄 So just use your github ID for it @eriawan